src/NUnitEngine/nunit.engine.api/nunit.engine.api.csproj

```diff
@@ -15,7 +15,7 @@
     <AssemblyTitle>NUnit Engine API ($(TargetFramework))</AssemblyTitle>
     <Description>Defines the interfaces used to access the NUnit Engine</Description>
     <AssemblyVersion>3.0.0.0</AssemblyVersion>
-    <FileVersion>$(ApiFileVersion)</FileVersion>
+    <FileVersion>3.99.0.0</FileVersion>
```

Comment:
I'm probably overlooking something, but why is it that this file has hardcoded `AssemblyVersion`/`FileVersion`? Is it because when we make a change here then we will manually update these versions?

Reply:
No. It's because we set AssemblyVersion to 3.0.0.0. If we set AssemblyVersion explicitly, then MSBuild uses that value for the FileVersion rather than the value we passed in the Version property. We can stop doing this in 4.0 if we can come up with a breakdown of API such that the version can be updated without breaking runners or extensions.